base/gcutils.jl

```diff
@@ -243,12 +243,21 @@ end
     GC.safepoint()
 
 Inserts a point in the program where garbage collection may run.
-This can be useful in rare cases in multi-threaded programs where some threads
-are allocating memory (and hence may need to run GC) but other threads are doing
-only simple operations (no allocation, task switches, or I/O).
-Calling this function periodically in non-allocating threads allows garbage
+
+Safepoint are fast, and do not themselves trigger garbage collection.
```

Comment:
`Safepoint are fast` seems a bit vague.

Perhaps include some expected number of cycles in a parentheses right after `Note that even though safepoints are fast`.

E.g. `Note that even though safepoints are fast (typically X-Y cycles)`.

IIRC @gbaraldi did some measurements about safepoint latency a while back and may have the numbers.

Reply:
A safepoint is always just a load.

Reply:
On my computer, it adds 5 `mov` instructions. Benchmarking a simple function shows it takes about 2 clock cycles, so I'm adding that